packages/contracts/utils/priceFeedInteractions.js

```diff
@@ -135,6 +135,6 @@
 //     Therefore, expected gas cost of a getLatestPrice() call is within a Trove function is (35k - 27.5k) 
 //     = 7500 gas upper bound.
     
-//     To check, deploy an instance of FunctionCaller contract to ropsten and mainnet, 
+//     To check, deploy an instance of FunctionCaller contract to rinkeby and mainnet, 
```

Comment:
Rinkeby is going to shut down in Q4 2022 we should probably switch to Goerli

Reply:
Resolved